flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -389,7 +396,22 @@ object AkkaUtils {
       ""
     }
 
-    ConfigFactory.parseString(configString + hostnameConfigString + sslConfigString).resolve()
+    val cookieConfigString = if(securityEnabled){
+      s"""
+         |akka {
+         |  remote {
+         |    require-cookie = $requireCookie
+         |    secure-cookie = "$secureCookie"
+         |  }
+         |}
+         """.stripMargin
+    }else{
```

Comment:
No need for the else block. We set set `require-cookie` to `off` in case it shouldn't be used